src/command_modules/azure-cli-container/azure/cli/command_modules/container/_validators.py

```diff
@@ -37,6 +42,14 @@ def validate_gitrepo_directory(ns):
         raise CLIError("The git repo directory cannot contain '..'")
 
 
+def validate_image(ns):
+    if ns.image.split(':')[0] in short_running_images and not ns.command_line:
+        logger.warning('Image "%s" has no long running process. The "--command-line" argument must be used to start a '
+                       'long running process inside the container for the container group to stay running. '
+                       'Ex: /bin/bash',
```

Comment:
could you get an aka.ms link from @jluk  that point to our documentation explain this in details?

Reply:
aka.ms/aci/troubleshoot points to our general doc, I can add in another header about this.